### DISCUSSION TITLE
MouseWheelZoom

### DISCUSSION THREAD

**Discussion by thomashdk** [Kortforsyningen/okapi#61] [Q&A]
Hi 

Is there a way to disable mouseWheelZoom ?? 

// Thomas

**Comment by iamfrank** [comment]
Hi @thomashdk 

Why would you want to disable mouseWheelZoom? Maybe to preserve scroll behaviour?
This can probably be configured in OpenLayers (the map engine beneath OKAPI), but I'm not sure this configuration is available in OKAPI.

**Reply by thomashdk** [reply]
I have a almost full width map, if ou scroll down the page and passes the map then it will start zoom out instead of scroll down the page.

**Reply by iamfrank** [reply]
Mousewheel zoom can be configured in OpenLayers - https://openlayers.org/en/latest/apidoc/module-ol_interaction.html#~DefaultsOptions
We'll look into making a new release of OKAPI, where this configuration is available.

**Comment by iamfrank** [comment]
I've created an issue regarding the configuration option: #64

**Reply by iamfrank** [reply]
Although a little belated, the option to configure mousewheel zoom is schedules to be released next week.
